<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1079</width>
    <height>762</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="board_graphicsview">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>10</y>
      <width>710</width>
      <height>621</height>
     </rect>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graph_graphicview">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>90</y>
      <width>135</width>
      <height>350</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="turn_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Player 1's Turn</string>
    </property>
   </widget>
   <widget class="QLabel" name="powerups_label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>160</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Powerups</string>
    </property>
   </widget>
   <widget class="QLabel" name="powerups_ForPlayerLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Player 1's Powerups:</string>
    </property>
   </widget>
   <widget class="QLabel" name="wins_label">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>30</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Wins: </string>
    </property>
   </widget>
   <widget class="QLabel" name="p1_num_wins_label">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>60</y>
      <width>21</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="p2_num_wins_label">
    <property name="geometry">
     <rect>
      <x>1030</x>
      <y>60</y>
      <width>21</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="turn_label_4">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>440</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Player 1</string>
    </property>
   </widget>
   <widget class="QLabel" name="turn_label_5">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>440</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Player 2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="left_button">
    <property name="geometry">
     <rect>
      <x>342</x>
      <y>640</y>
      <width>201</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>&lt;&lt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="right_button">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>640</y>
      <width>211</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>&gt;&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="drop_button">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>670</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>DROP!</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="pwrup_graphicsview">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>60</width>
      <height>121</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="nullpiece_label">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>240</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
     </font>
    </property>
    <property name="text">
     <string>Nullpiece</string>
    </property>
   </widget>
   <widget class="QLabel" name="nullpiece_qty_label">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>260</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>x2</string>
    </property>
   </widget>
   <widget class="QLabel" name="convert_label">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>310</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
     </font>
    </property>
    <property name="text">
     <string>ConvertBelow</string>
    </property>
   </widget>
   <widget class="QLabel" name="cnvrt_below_qty_label">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>330</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>x2</string>
    </property>
   </widget>
   <widget class="QLabel" name="turn_label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>360</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>(Click on the icon to activate)</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1079</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuADVANCED_connect_four">
    <property name="title">
     <string>ADVANCED connect four!</string>
    </property>
   </widget>
   <addaction name="menuADVANCED_connect_four"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
